dnf5.spec

```diff
@@ -21,6 +21,12 @@ Provides:       microdnf = %{version}-%{release}
 Obsoletes:      microdnf < 4
 %endif
 
+Provides:       dnf = %{version}-%{release}
+Obsoletes:      dnf < 5
+
+Provides:       %{name}-yum = %{version}-%{release}
+Obsoletes:      %{name}-yum < 5
```

Comment:
Are you sure about this? I guess that it will obsolete dnf5-yum and this package is not in distribution. I guess that 
correct would be `Obsoletes:      dnf-yum < 5`. Or did I miss something?